sql-db/sql-app/src/main/java/io/quarkus/ts/sqldb/sqlapp/Book.java

```diff
@@ -1,14 +1,24 @@
 package io.quarkus.ts.sqldb.sqlapp;
 
 import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
 import jakarta.persistence.Table;
 import jakarta.validation.constraints.NotBlank;
 
-import io.quarkus.hibernate.orm.panache.PanacheEntity;
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
 
 @Entity
 @Table(name = "book")
-public class Book extends PanacheEntity {
+public class Book extends PanacheEntityBase {
+
+    @Id
+    @SequenceGenerator(name = "bookSequence", sequenceName = "Book_SEQ", allocationSize = 1, initialValue = 8)
```

Comment:
`initialValue = 8` is a hack to make tests happy :) But questionable for real life application imho

Reply:
real life application can't avoid duplicate index exception, if they want to migrate data (or initialize like we do) they need to move initial value. It's highly unlikely that real life application is going to use unsigned primary key. So I don't know whay doyou suggest?

Reply:
from practice - what we did was offset (about 10000 + ), but I don't see a difference to 8

Reply:
IIRC sequence strategy should detect current value (expect when they ask for ids every xy values), so I can play with it if you like; I just don't see what's wrong about initial value.

Reply:
Ok, it calls db anyway, I'll fix it.